R/derive_extreme_event.R

```diff
@@ -475,6 +493,15 @@ derive_extreme_event <- function(dataset = NULL,
   )
   selected_records <- bind_rows(selected_records_ls)
 
+  # Check for duplicates
+  signal_duplicate_records(
+    dataset = selected_records,
+    by_vars = append(by_vars, order),
+    msg = paste("Check duplicates: selected_records dataset contains duplicate records
```

Comment:
The users don't know the `selected_records` dataset because it is an internal dataset. We need to use a description like "the dataset which consists of all records selected for any of the events defined by `events`".